packages/client/src/components/BigMovieCard/BigMovieCard.jsx

```diff
@@ -250,6 +254,7 @@ export const BigMovieCard = ({ currentMovie, currentReview, user }) => {
           />
         </CardContent>
       </Card>
+      <PopUpModulMenu currentMovie={currentMovie} />
```

Comment:
Maybe AddToCartConfirmation or something like that would be better? But not necessarily :D

Reply:
thats a good suggestion!